lib/msal-browser/docs/working-with-b2c.md

```diff
@@ -0,0 +1,103 @@
+# Working with MSAL.js and Azure AD B2C
+
+> :warning: Before you start here, make sure you understand [how to initialize an app object](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/initialization.md) and [working with resources and scopes](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/resources-and-scopes.md). We also recommend general familiarity with **Azure AD B2C**. See the [B2C documentation](https://docs.microsoft.com/azure/active-directory-b2c/technical-overview) for more.
+
+**MSAL.js supports** authentication with **social** (Microsoft, Google, Facebook etc.) and **local** (Azure AD work and school) identities using **Azure AD B2C** (*B2C* for short). When developing B2C apps with **MSAL.js**, there are a few important details to keep in mind.
+
+## Quick Facts
+
+With B2C:
+
+- You **can** authenticate users with social identities.
+- You **can** authorize users to access your **B2C protected** resources.
+- You **cannot** obtain tokens for Microsoft APIs (e.g. MS Graph API) using [delegated permissions](#b2c-and-delegated-permissions).
+- You **can** obtain tokens for Microsoft APIs using [application permissions](#b2c-and-application-permissions) (management scenarios).
+
+## B2C App Configuration
+
+The following is a B2C app configuration example:
+
+```javascript
+const msalConfig = {
+  auth: {
+    clientId: "<your-clientID>",
+    authority: "https://<your-tenant>.b2clogin.com/<your-tenant>.onmicrosoft.com/<your-policyID>",
+    validateAuthority: true,
```

Comment:
Since this is a msal-browser doc we should remove `validateAuthority` as it is no longer a config option in 2.0

Reply:
Aah, of course -removed